research/picai/fedavg/run_fl_single_node.slrm

```diff
@@ -71,25 +66,46 @@ SERVER_OUTPUT_FILE="${SERVER_LOG_DIR}server.out"
 echo "Server logging at: ${SERVER_OUTPUT_FILE}"
 echo "Launching Server"
 
-nohup python -m research.picai.fedavg.server --config_path ${SERVER_CONFIG_PATH} --artifact_dir ${SERVER_LOG_DIR} --n_clients ${N_CLIENTS} > ${SERVER_OUTPUT_FILE} 2>&1 &
-
-# Sleep for 20 seconds to allow the server to come up.
-sleep 20
-
-# Start n number of clients and divert the outputs to their own files
-END=$(($N_CLIENTS - 1))
-for i in $(seq 1 $END);
-do
-    CLIENT_NAME="client_${i}"
-    echo "Launching ${CLIENT_NAME}"
-
-    CLIENT_LOG_PATH="${CLIENT_LOG_DIR}client_${i}.out"
-    echo "${CLIENT_NAME} logging at: ${CLIENT_LOG_PATH}"
-    nohup python -m research.picai.fedavg.client --artifact_dir ${CLIENT_LOG_DIR} > ${CLIENT_LOG_PATH} 2>&1 &
-done
 
-echo "FL Processes Running"
+echo "Finished FL Processes"
```

Comment:
Will this immediately echo this message? I wonder if this should be moved to the bottom of this script after the final wait?

Reply:
Yeah good catch!